apps/back-end/src/services/Dataset.ts

```diff
@@ -111,10 +111,19 @@ export class Dataset {
     fs.createReadStream(path.join(this.folderPath, "locations.json"), "utf8");
 
   getTotals = () => {
-    const totals = {};
-
-    this.searchablePropValues.forEach((itemValues, itemIx) => {
-      console.log(itemValues, itemIx)
+    const totals: any = {
+      any: 0
+    };
+
+    this.searchablePropValues.forEach((itemValues) => {
+      if (totals[itemValues[1] as string]) {
```

Comment:
This doesn't seem to deal with the possibility that `itemValues[1]` might be an `string[]`? Or that the array might have length 0?

The types are pesky. But this seems to keep the checker happy:
```
    this.searchablePropValues.forEach((itemValues) => {
      const label = itemValues[1];
      if (typeof label !== "string") return;
      totals[label] = 1 + (totals[label] ?? 0);
      totals.any = 1 + (totals.any ?? 0);
    });
```